priiloader/source/main.cpp

```diff
@@ -25,6 +25,7 @@ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 #define MAGIC_WORD_DACO 1
 #define MAGIC_WORD_PUNE 2
 #define MAGIC_WORD_ABRA 3
+#define MAGIC_TITLE_ID 0x505249494C4F4144llu // 'PRII-LOAD'

```

Comment:
what is this title id about?

Reply:
When booting into this title id from Wii U mode the Priiloader menu will be opened. That way you can boot directly into the Priiloader config menu using a Wii U homebrew app without the need for a USB keyboard.
See `tools/WiiUForwarder`.